### ISSUE TITLE
Uncaught TypeError: moment.locale is not a function

### ISSUE THREAD

**Issue by gregorysl** [ant-design/ant-design#8876]
### Version
3.1.1

### Environment
Windows 10, Chrome 63

### Reproduction link
[https://github.com/ant-design/parcel-antd](https://github.com/ant-design/parcel-antd)

### Steps to reproduce
1. Change index.js of parcel sample to:
```
import React from 'react';
import ReactDOM from 'react-dom';
import { Button } from 'antd';
import { LocaleProvider } from 'antd';
// The default locale is en-US, but we can change it to other language
import frFR from 'antd/lib/locale-provider/fr_FR';
import moment from 'moment';
import 'moment/locale/fr';

ReactDOM.render(
  <LocaleProvider locale={frFR}>
  <Button type="primary">Button</Button>
  </LocaleProvider>
, document.getElementById('mountNode'));
```
2. Run app

### What is expected?
Application will run successfully with locale loaded

### What is actually happening?
Error thrown in console
parcel-antd.js:27172 Uncaught TypeError: moment.locale is not a function
    at Object.require.6.moment (parcel-antd.js:27172)
    at newRequire (parcel-antd.js:42)
    at localRequire (parcel-antd.js:48)
    at Object.require.4.antd/es/locale-provider (parcel-antd.js:31853)
    at newRequire (parcel-antd.js:42)
    at require.183 (parcel-antd.js:67)
    at parcel-antd.js:72

**Comment by yesmeck** [comment]
https://github.com/parcel-bundler/parcel/issues/418

**Comment by yesmeck** [comment]
Close since it's not issue of antd.

**Comment by bytewiz** [comment]
@gregorysl How did you solve this issue, I am facing the same

**Comment by gregorysl** [comment]
I was using this fork: https://github.com/parcel-bundler/parcel/pull/530

**Comment by fathyb** [comment]
@gregorysl @bytewiz the PR parcel-bundler/parcel#530 will not be merged, however I created #9594 to fix it without the Parcel PR. Feel free to try it and confirm me if it works for you.
